.travis.yml

```diff
@@ -14,4 +14,4 @@ cache:
     - $HOME/.gradle/wrapper/
 
 script:
-  - ./gradlew build
+  - ./gradlew build test
```

Comment:
AFAIK tests are run without specifying it...
At least in my project it works like that 100%
Please, check that this change is needed

Reply:
You were right :
gradlew build --dry-run
:minifyPlayJavaScript SKIPPED
:createPlayAssetsJar SKIPPED
:compileKotlin SKIPPED
:compileJava SKIPPED
:compilePlayRoutes SKIPPED
:compilePlayTwirlTemplates SKIPPED
:compileScala SKIPPED
:processResources SKIPPED
:classes SKIPPED
:inspectClassesForKotlinIC SKIPPED
:jar SKIPPED
:createMainDistributionJar SKIPPED
:createMainStartScripts SKIPPED
:distTar SKIPPED
:distZip SKIPPED
:assemble SKIPPED
:compileTestKotlin SKIPPED
:compileTestJava SKIPPED
:compileTestScala SKIPPED
:processTestResources SKIPPED
:testClasses SKIPPED
**:test SKIPPED**
:check SKIPPED
:build SKIPPED